manager/assets/modext/widgets/system/modx.panel.context.js

```diff
@@ -54,6 +54,12 @@ MODx.panel.Context = function(config) {
 					,width: 300
 					,grow: true
 				},{
+					xtype: 'numberfield'
+					,fieldLabel: _('rank')
+					,name: 'rank'
+					,allowBlank: false
```

Comment:
No, because of the [DB schema](https://github.com/modxcms/revolution/blob/2.x/core/model/schema/modx.mysql.schema.xml#L451)